salt/utils/event.py

```diff
@@ -1075,24 +1075,39 @@ def _handle_signals(self, signum, sigframe):
         super(EventReturn, self)._handle_signals(signum, sigframe)
 
     def flush_events(self):
-        event_return = '{0}.event_return'.format(
-            self.opts['event_return']
-        )
+        if isinstance(self.opts['event_return'], list):
+            # Multiple event returners
+            for r in self.opts['event_return']:
+                if log.level <= logging.DEBUG:
+                    log.debug('Calling event returner {0}, one of many.'.format(r))
+                event_return = '{0}.event_return'.format(r)
+                self._flush_event_single(event_return)
+        else:
+            # Only a single event returner
+            if log.level <= logging.DEBUG:
```

Comment:
We shouldn't need to do a check like this here. The logging module will handle figuring out what the log level is and whether or not the message should be routed to a handler.